p-token/src/processor/close_account.rs

```diff
@@ -9,6 +9,8 @@ use token_interface::{
 use super::validate_owner;
 
 /// Incinerator (`1nc1nerator11111111111111111111111111111111`) address.
+/// JC nit: any reason to not use the one defined in the interface instead? Or
+/// is this defined elsewhere in the SDK?
```

Comment:
I think it is only defined on `solana-ids`. At the time I don't think the SDK was broken-up. We could switch to use the `solana-ids` crate now, the only potential drawback is that it brings a few dependencies to convert the string to a base58 value – I guess this is ok since the conversion happens at compile time.

Reply:
I was referring to the one defined at https://github.com/solana-program/token/blob/7433164b2ac60aeb3305016575ec308b0df6555e/interface/src/state/account.rs#L6

Reply:
Oh, you are right. Updated to use that constant instead.